metadata-db-app/src/cmr/metadata_db/data/oracle/generic_documents.clj

```diff
@@ -41,23 +39,49 @@
                           :schema schema
                           :format format ;; concepts convert this to mimetype in the get, but we already have mimetype
                           :mime-type mime_type
-                          :metadata (when metadata (cutil/gzip-blob->string metadata))
                           :revision-id (int revision_id)
                           :revision-date (oracle/oracle-timestamp->str-time db revision_date)
                           :created-at (when created_at
                                         (oracle/oracle-timestamp->str-time db created_at))
                           :deleted (not= (int deleted) 0)
                           :user-id user_id
-                          :transaction-id transaction_id}))
+                          :transaction-id transaction_id}
+                          :metadata (when metadata (json/parse-string (cutil/gzip-blob->string metadata) true))))
```

Comment:
why are we parsing a json string after it has been zipped up?

Reply:
gzip->string is unzipping the data from the database